insights/parsers/lspci.py

```diff
@@ -77,21 +79,23 @@ def parse_content(self, content):
             scanner(self)
         # Parse kernel driver lines
         self.data = {}
-        bus_device_function = None
-        bus_device_function_re = re.compile(r'[0-9a-f]+:[0-9a-f]+.[0-9a-f]+')
+        slot = None
+        solt_re = re.compile(r'^[0-9a-f]+:[0-9a-f]+.[0-9a-f]+')
```

Comment:
@xiangce I think you meant to use `slot_re` here and below?

Reply:
@bfahr - Fixed. Thanks for reviewing.